client/types/disputes.d.ts

```diff
@@ -51,7 +64,7 @@ export interface Dispute {
 	evidence: Evidence;
 	fileSize?: Record< string, number >;
 	reason: DisputeReason;
-	charge: Charge;
+	charge: Charge | string;
```

Comment:
When the `Dispute` is nested within a `Charge`, the charge property is a charge id string (`ch_`) rather than a dispute object.